From: Helmut Grohne <helmut@subdivi.de>
Date: Fri, 14 Oct 2016 08:14:02 +0200
Subject: Use CC from environment

Fix ispell failure to cross build from source, because the build
system overrides the maintainer supplied cross compiler with the
build architecture compiler.

Bug-Debian: https://bugs.debian.org/840715
---
 Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 09e5707..0d0fcb3 100644
--- a/Makefile
+++ b/Makefile
@@ -536,7 +536,7 @@ defhash.h:  config.X local.h Makefile
 
 config.sh:  config.X defhash.h local.h Makefile
 	set $(SHELLDEBUG); \
-	for var in BAKEXT BINDIR CC COUNTSUFFIX DEFDICT DEFHASH \
+	for var in BAKEXT BINDIR COUNTSUFFIX DEFDICT DEFHASH \
 	  DEFLANG EXEEXT HASHSUFFIX INSTALL \
 	  LANGUAGES LIBDIR LIBES LINK LINT LINTFLAGS LOOK_XREF \
 	  MAKE_SORTTMP MAN1DIR MAN1EXT MAN45DIR MAN45EXT MAN45SECT MASTERHASH \
@@ -548,6 +548,7 @@ config.sh:  config.X defhash.h local.h Makefile
 	      | sed -e 's/"[^"]*$$/'"'/" -e "s/=/='/" -e 's/\\"/"/g' \
 	      | sed -n -e '$$p'; \
 	  done > config.sh; \
+	echo "CC='$(CC)'" >> config.sh; \
 	echo "CFLAGS='$(CFLAGS)'" >> config.sh; \
 	echo 'case "$$MAKE_SORTTMP" in "") \
 	  SORTTMP="-e /!!SORTTMP!!/s/=.*$$/=/";; *) SORTTMP=;; esac' \
